synfig-studio/src/gui/canvasview.cpp

```diff
@@ -298,6 +302,26 @@ class studio::CanvasViewUIInterface : public UIInterface
 	virtual bool
 	amount_complete(int current, int total)
 	{
+		float temp = (float)current/(float)total;
+		
+		if(temp >= 1.0)
+		{
+			view->statusbar->show();
+			view->progressbar->hide();
+		}
+
+		if(fabs(temp-cur_progress) >= 0.01)
```

Comment:
Also, should it be `else if` instead of `if`?

If progress value changes from 1.1 to 1.2, then this condition will be triggered and progressbar will appear. ^__^

Reply:
well I believe this should never occur as 1.0 is the value that should be triggered for completion.
However adding this will eliminate any chances of error.